app/components/build/AvatarWithUnknownEmailPrompt.js

```diff
@@ -114,49 +117,90 @@ class AvatarWithUnknownEmailPrompt extends React.PureComponent {
       return {};
     }
 
+    const { email: authorEmail } = this.props.build.createdBy;
```

Comment:
Woah, this syntax is interesting! I think I like it?

Is there any benefit over:

```js
const authorEmail = this.props.build.createdBy.email
```